notebooks/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 BSplineKit = "093aae92-e908-43d7-9660-e50ee39d5a0a"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
```

Comment:
This shouldn't be needed either, ForwardDiff is only needed in the main project.